committeeoversightapp/models.py

```diff
@@ -185,7 +185,12 @@ def __str__(self):
         return self.rating
 
 
-class StaticPage(Page):
+class ResetMixin(object):
```

Comment:
@jeancochrane what do you think of this mixin approach?

Reply:
Really nice 👍